src/EPR.SubsidiaryBulkUpload.Application/Services/BulkSubsidiaryProcessor.cs

```diff
@@ -174,4 +233,36 @@ private async Task AddSubsidiary(OrganisationResponseModel parent, OrganisationR
 
         _logger.LogInformation("Subsidiary Company {SubsidiaryReferenceNumber} {SubsidiaryName} linked to {ParentReferenceNumber} in the database.", subsidiary.referenceNumber, subsidiary.name, parent.referenceNumber);
     }
+
+    private async Task ProcessFranchisee(IEnumerable<CompaniesHouseCompany> subsidiaries, OrganisationResponseModel parentOrg, UserRequestModel userRequestModel)
+    {
+        // companies with franchisee flag.
+        var companiesWithFranchiseeFlagRecords = subsidiaries.Where(ch => !string.IsNullOrEmpty(ch.franchisee_licensee_tenant) && string.Equals(ch.franchisee_licensee_tenant, "Y", StringComparison.OrdinalIgnoreCase))
```

Comment:
Flag can only be empty or Y - you can just do  `subsidiaries.Where(ch => ch.franchisee_licensee_tenant, == "Y")`

Reply:
this condition is to be Not empty and Y.